docs/notifications.md

```diff
@@ -8,19 +8,21 @@ receive them as JSON-RPC notifications from the server.
 
 ## Events
 Currently supported events:
- * new block added
-   Contents: block.
+ * new block added<br>
+   Contents: block.<br>
```

Comment:
Can we avoid this `<br>`? Format it all in a different way? It will improve "rendered" MD, but I'd like for it to be readable without rendering as well.